core/api/src/services/ledger/facade/volume.ts

```diff
@@ -71,6 +73,34 @@ const TxVolumeAmountSinceFactory = () => {
             $and: [{ timestamp: { $gte: timestamp } }],
           },
         },
+        {
+          $lookup: {
```

Comment:
Not sure if this might have performance implications. Two things we can do:
- add indices for `_journal` (and `_original_journal`?)
- switch to using datetime from `journals` collection instead of `medici_transactions` collection

Reply:
Update: `_journal` is already indexed by medici (see [here](https://github.com/flash-oss/medici/blob/master/src/models/transaction.ts#L75)), and we can alternatively change the lookup collection later on if we notice performance issues